### PR TITLE
[site-wide] corrects 'company' to 'corporation' where relevant

### PR DESCRIPTION
**Pull request by philcampeau**
## Before clicking "Create"

- [x] Branch is derived from the latest `master`
- [x] Add the `pins` or `archive_alpha_games` label if this change will break existing games
- [x] Code passes linter with `docker compose exec rack rubocop -a`
- [x] Tests pass cleanly with `docker compose exec rack rake`

### Explanation of Change

I went through all the private company descriptions and changed the word company to corporation where applicable to keep in line with the terminology on the site.

No changes to game code, only text descriptions.

### CODE REVIEW

**Review by tobymao — APPROVED** [review]

**Comment by ollybh** [comment]
I'm not sure this is a good idea. Corporation is the internal 18xx.games term for the type of entity that has multiple shares and a stock market price, but many rulebooks call this a company.

**Comment by philcampeau** [comment]
> I'm not sure this is a good idea. Corporation is the internal 18xx.games term for the type of entity that has multiple shares and a stock market price, but many rulebooks call this a company.

@ollybh Many rulebooks use a lot of random terms for various entities. But at least on this site, isn't it better to have uniform terminology?

**Comment by tobymao** [comment]
@ollybh 18xx.games has standardized the terminology

**Comment by philcampeau** [comment]
On top of that, in a lot of cases these terms are used interchangeably even within the same game files.

File: lib/engine/game/g_1817_wo/entities.rb [hunk 1]
```diff
@@ -32,7 +32,7 @@ module Entities
             name: 'Mountain (Ocean) Engineers',
             value: 40,
             revenue: 0,
-            desc: 'Owning company receives $20 after laying a yellow tile in a '\
+            desc: 'Owning corp receives $20 after laying a yellow tile in a '\
```
Comment (perwestling) [inline lib/engine/game/g_1817_wo/entities.rb:35]: A mix of corp/corporation. Use corporation here as well?

File: lib/engine/game/g_1828/entities.rb [hunk 2]
```diff
@@ -96,8 +96,8 @@ module Entities
             value: 120,
             revenue: 20,
             desc: 'Blocks Adrian & Ann Arbor (E7) while owned by a player. ' \
-                  'A yellow track tile is placed at E7 when purchased by a company. ' \
-                  'Owning company may (once) place one additional yellow track tile ' \
+                  'A yellow track tile is placed at E7 when purchased by a corporation. ' \
+                  'Owning corp may (once) place one additional yellow track tile ' \
```
Comment (perwestling) [inline lib/engine/game/g_1828/entities.rb:100]: corp -> corporation?

File: lib/engine/game/g_1832/entities.rb [hunk 3]
```diff
@@ -232,7 +232,7 @@ module Entities
             revenue: 10,
             desc: 'This company has a $10 token which may be placed in any non-coastal city'\
                   ' (Atlantic or Gulf Coast) as an extra token lay during the'\
-                  ' token placement step of an owning public company\'s operating round.',
+                  ' token placement step of an owning corp\'s operating round.',
```
Comment (perwestling) [inline lib/engine/game/g_1832/entities.rb:235]: Mix of corp and corporation in this file.

File: lib/engine/game/g_1829/entities.rb [hunk 4]
```diff
@@ -48,39 +48,39 @@ module Entities
             type: 'steam',
             value: 315,
             revenue: 25,
-            desc: 'Steam. pays 40$ to company with token in B17',
+            desc: 'Steam. Pays £40 to corp with token in B17',
```
Comment (perwestling) [inline lib/engine/game/g_1829/entities.rb:51]: corp -> corporation
?

File: lib/engine/game/g_1860/entities.rb [hunk 5]
```diff
@@ -9,7 +9,7 @@ module Entities
             name: 'Brading Harbour Company',
             value: 30,
             revenue: 5,
-            desc: 'Can be exchanged for a share in the BHI&R public company',
+            desc: 'Can be exchanged for a share in the BHI&R corporation',
```
Comment (perwestling) [inline lib/engine/game/g_1860/entities.rb:12]: To be consistent, add a "." at the end.

File: lib/engine/game/g_1868_wy/entities.rb [hunk 6]
```diff
@@ -43,7 +43,7 @@ def self.def_corporation(**kwargs)
               {
                 type: 'base',
                 description: 'Credit Mobilier',
-                desc_detail: 'After yellow track is laid by any Railroad Company in the Credit Mobilier region (row J and south, column 23 and west, outlined in brown) which either extends track west from Omaha or east from Ogden, UP shareholders are paid the total terrain costs of the hex as dividends, even if the track was laid by the Railroad Company for free via a private company ability. Credit Mobilier closes at phase 5, or when the Golden Spike bonus is collected, whichever comes first.',
+                desc_detail: 'After yellow track is laid by any Corporation in the Credit Mobilier region (row J and south, column 23 and west, outlined in brown) which either extends track west from Omaha or east from Ogden, UP shareholders are paid the total terrain costs of the hex as dividends, even if the track was laid by the Corporation for free via a private company ability. Credit Mobilier closes at phase 5, or when the Golden Spike bonus is collected, whichever comes first.',
```
Comment (perwestling) [inline lib/engine/game/g_1868_wy/entities.rb:46]: Corporation -> corporation
?

File: lib/engine/game/g_1868_wy/entities.rb [hunk 7]
```diff
@@ -80,7 +80,7 @@ def self.def_corporation(**kwargs)
               {
                 type: 'base',
                 description: 'Purple Home Hex',
-                desc_detail: 'No other Railroad Company may run routes to Chadron (G27).',
+                desc_detail: 'No other Corporation may run routes to Chadron (G27).',
```
Comment (perwestling) [inline lib/engine/game/g_1868_wy/entities.rb:83]: Corporation -> corporation
?

File: lib/engine/game/g_1868_wy/entities.rb [hunk 8]
```diff
@@ -157,7 +157,7 @@ def self.def_corporation(**kwargs)
             desc: 'Buyer or auction winner chooses one of Wylie Permanent Camping Company '\
                   '($10 bonus for running routes to Yellowstone--C5 or D4), Trabing Brothers Frontier Fort Resupply '\
                   '($10 bonus for running routes to Forts), or Midwest Oil Refinery '\
-                  '($10 bonus for any train owned by any Railroad Company running a route to Casper--H18)',
+                  '($10 bonus for any train owned by any Corporation running a route to Casper--H18)',
```
Comment (perwestling) [inline lib/engine/game/g_1868_wy/entities.rb:160]: Corporation -> corporation

?

Comment (perwestling) [reply lib/engine/game/g_1868_wy/entities.rb:160]: There are more occurance of this in this file.

File: lib/engine/game/g_1870/entities.rb [hunk 9]
```diff
@@ -17,14 +17,14 @@ module Entities
...
+                  ' River. A corporation may lay track along the river, but may not lay track to cross the river, or do an'\
+                  ' upgrade that would cause track to cross the river. The corporation that purchases the Mississippi'\
                   ' River Bridge Company may build in one of the hexes along the Mississippi River for a $40 discount.'\
-                  ' This company may be purchased by one of the two companies on the Mississippi River (Missouri Pacific'\
-                  ' or St.Louis Southwestern) in phase one for $20 to $40. If one of these two public companies purchases'\
-                  ' this private company during their first operating round, that company can lay a tile at its starting'\
-                  ' city for no cost and in addition to its normal tile lay(s). The company cannot lay a tile in their'\
+                  ' This company may be purchased by one of the two corps on the Mississippi River (Missouri Pacific'\
+                  ' or St.Louis Southwestern) in phase one for $20 to $40. If one of these two corps purchases'\
```
Comment (perwestling) [inline lib/engine/game/g_1870/entities.rb:25]: corps -> corporations

File: lib/engine/game/g_1870/entities.rb [hunk 10]
```diff
@@ -17,14 +17,14 @@ module Entities
...
+                  ' this private company during their first operating round, that corporation can lay a tile at its starting'\
+                  ' city for no cost and in addition to its normal tile lay(s). The corp cannot lay a tile in their'\
```
Comment (perwestling) [inline lib/engine/game/g_1870/entities.rb:27]: corp -> corporation

File: lib/engine/game/g_1870/entities.rb [hunk 11]
```diff
@@ -80,10 +80,10 @@ module Entities
             desc: 'This company has two tokens. One represents an open port and the other is a closed port. One (but'\
                   ' not both) of these tokens may be placed on one of the cities: Memphis (H17), Baton Rouge (M14), Mobile'\
                   ' (M20), Galveston (N7) and New Orleans (N17). Either token increases the value of the city for the owning'\
-                  ' company by $20. The open port token also increases the value of the city for all other companies by $10.'\
-                  ' If the president of the owning company places the closed port token, the private company is closed. If'\
-                  ' the open port token is placed, it may be replaced in a later operating round by the closed port token,'\
-                  ' closing the company.',
+                  ' corporation by $20. The open port token also increases the value of the city for all other corporations by'\
+                  ' $10. If the president of the owning corporation places the closed port token, the private company is'\
```
Comment (perwestling) [inline lib/engine/game/g_1870/entities.rb:84]: private company -> company

File: lib/engine/game/g_1870/entities.rb [hunk 12]
```diff
@@ -107,8 +107,8 @@ module Entities
             value: 140,
             revenue: 0,
             desc: "This is the President's certificate of the St.Louis-San Francisco Railway. The purchaser sets the"\
-                  ' par value of the railway. Unlike other companies, this company may operate with just 20% sold. It may'\
-                  ' not be purchased by another public company.',
+                  ' par value of the railway. Unlike other corporations, the St.Louis-San Francisco Railway may operate'\
+                  ' with just 20% sold. This private company may not be purchased by a corporation.',
```
Comment (perwestling) [inline lib/engine/game/g_1870/entities.rb:111]: private company -> company

File: lib/engine/game/g_1888/entities.rb [hunk 13]
```diff
@@ -17,7 +17,7 @@ module Entities
             name: 'Terracotta Army',
             value: 50,
             revenue: 10,
-            desc: 'Comes with the red Xi’an tile which shows higher value for this off-board location. A company owning this private may lay this tile in addition to it’s normal tile lay step. No connection is required. This does not close the company. If it wasn’t laid when the company closes the red tile is out of the game.',
+            desc: 'Comes with the red Xi’an tile which shows higher value for this off-board location. A corporation owning this private may lay this tile in addition to it’s normal tile lay step. No connection is required. This does not close the company. If it wasn’t laid when the company closes the red tile is out of the game.',
```
Comment (perwestling) [inline lib/engine/game/g_1888/entities.rb:20]: private -> company

File: lib/engine/game/g_1888/entities.rb [hunk 14]
```diff
@@ -35,7 +35,7 @@ module Entities
...
             revenue: 15,
-            desc: 'Comes with the yellow mountain tile.  A company owning this private may lay the mine on hex C5 in addition to it’s normal tile laying step. No connection is required. If it wasn’t laid when the company closes the mountain tile is out of the game. If a train runs to or through the mine the company treasury gets ¥40 from the bank. It counts against the range of a train.  Any company may lay a normal tile on hex C5 after this private company was sold to a company or closed at the cost of ¥40. Then the yellow mountain tile can’t be placed.',
+            desc: 'Comes with the yellow mountain tile.  A corporation owning this private may lay the mine on hex C5 in addition to it’s normal tile laying step. No connection is required. If it wasn’t laid when the company closes the mountain tile is out of the game. If a train runs to or through the mine the owning corporation\'s treasury gets ¥40 from the bank. It counts against the range of a train.  Any corporation may lay a normal tile on hex C5 after this private company was sold to a corporation or closed at the cost of ¥40. Then the yellow mountain tile can’t be placed.',
```
Comment (perwestling) [inline lib/engine/game/g_1888/entities.rb:38]: private company -> company

File: lib/engine/game/g_1888/entities.rb [hunk 15]
```diff
@@ -53,7 +53,7 @@ module Entities
             name: 'Great Wall of China',
             value: 100,
             revenue: 20,
-            desc: 'A company owning this private may lay one or two yellow tiles for free onto hexes with the Great Wall in addition to the normal tile lay step. No connection is required. This does not close the company. If laying two tiles, both tiles must be laid during the same turn and must be connected.',
+            desc: 'A corporation owning this private may lay one or two yellow tiles for free onto hexes with the Great Wall in addition to the normal tile lay step. No connection is required. This does not close the company. If laying two tiles, both tiles must be laid during the same turn and must be connected.',
```
Comment (perwestling) [inline lib/engine/game/g_1888/entities.rb:56]: private -> company

File: lib/engine/game/g_1888/entities.rb [hunk 16]
```diff
@@ -71,7 +71,7 @@ module Entities
             name: 'Yanda Railway Ferry',
             value: 125,
             revenue: 25,
-            desc: 'Comes with the blue ferry tiles. A company owning this private may lay these tiles in addition to it’s normal tile laying step on both of the Dalian and Yantai hexes. No connection is required. If they are not laid when the company closes the ferry tiles are out of the game.',
+            desc: 'Comes with the blue ferry tiles. A corporation owning this private may lay these tiles in addition to it’s normal tile laying step on both of the Dalian and Yantai hexes. No connection is required. If they are not laid when the company closes the ferry tiles are out of the game.',
```
Comment (perwestling) [inline lib/engine/game/g_1888/entities.rb:74]: private -> company

**Comment by philcampeau** [comment]
Good to go!

File: lib/engine/game/g_1888/entities.rb (hunk 16 above)
Comment (perwestling) [inline lib/engine/game/g_1888/entities.rb:74]: it's -> its

File: lib/engine/game/g_1888/entities.rb [hunk 17]
```diff
@@ -92,7 +92,7 @@ module Entities
             name: 'Forbidden City',
             value: 150,
             revenue: 30,
-            desc: 'A player owning this private company may exchange it into an IPO share of a company with a token in Beijing (at the time of exchange). If sold to a company this ability is lost.',
+            desc: 'A player owning this private company may exchange it into an IPO share of a corporation with a token in Beijing (at the time of exchange). If sold to a corporation this ability is lost.',
```
Comment (perwestling) [inline lib/engine/game/g_1888/entities.rb:95]: private company -> company

File: lib/engine/game/g_18_ardennes/entities.rb [hunk 18]
```diff
@@ -338,7 +338,7 @@ module Entities
         def company_header(company)
           case company.type
           when :minor then 'MINOR COMPANY'
```
Comment (perwestling) [inline lib/engine/game/g_18_ardennes/entities.rb:340]: Should it be MINOR COMPANY, MINOR CORPORATION or just MINOR?

File: lib/engine/game/g_18_ardennes/game.rb [hunk 19]
```diff
@@ -21,7 +21,7 @@ class Game < Game::Base
         include Tiles
         include Trains
 
-        # Minors that have been pledged in bids to start public companies.
+        # Minors that have been pledged in bids to start corporations.
         # Used by {Step::MajorAuction} to pass this information to
         # {Step::BuySellParSharesCompanies}. This is a hash whose keys are
         # the major corporations and the values are the minor corporations.
```
Comment (perwestling) [inline lib/engine/game/g_18_ardennes/game.rb:27]: Is it "major corporations" or just corporations?

File: lib/engine/game/g_18_ardennes/market.rb [hunk 20]
```diff
@@ -16,7 +16,7 @@ module Market
         ).freeze
         MARKET_TEXT = Base::MARKET_TEXT.merge(
           par_1: 'Minor company starting values',
```
Comment (perwestling) [inline lib/engine/game/g_18_ardennes/market.rb:18]: Is "Minor company" correct?

File: lib/engine/game/g_18_ardennes/market.rb [hunk 21]
```diff
@@ -16,7 +16,7 @@ module Market
...
-          par_2: 'Major company starting values',
+          par_2: 'Corporation starting values',
           repar: 'Minor companies cannot merge'
```
Comment (perwestling) [inline lib/engine/game/g_18_ardennes/market.rb:20]: Same comment here.

File: lib/engine/game/g_18_ardennes/step/major_auction.rb [hunk 22]
```diff
@@ -21,16 +21,16 @@ def actions(entity)
...
           def setup
             setup_auction
             @concessions = @game.companies.select do |company|
               company.type == :concession && company.owner.nil? && !company.closed?
             end
-            # Hash showing which minors can be used to start a public company.
-            # The public companies are the has keys, each value is an array of
+            # Hash showing which minors can be used to start a corporation.
+            # The corporations are the hash keys, each value is an array of
             # minor companies.
```
Comment (perwestling) [inline lib/engine/game/g_18_ardennes/step/major_auction.rb:34]: minor companies -> minors
 
 ?

File: lib/engine/game/g_18_ardennes/step/major_auction.rb [hunk 23]
```diff
@@ -152,9 +152,9 @@ def concession_corporation(concession)
             @game.corporation_by_id(concession.id)
           end
 
-          # If no public companies have yet been started then there are
+          # If no corporations have yet been started then there are
           # geographical restrictions on which minor companies can be used to
```
Comment (perwestling) [inline lib/engine/game/g_18_ardennes/step/major_auction.rb:156]: Another "minor companies "

File: lib/engine/game/g_18_bf/trains.rb [hunk 24]
```diff
@@ -24,7 +24,7 @@ module Trains
           'minors_convert' => ['Minors convert',
                                'Minor companies may merge, convert or be taken over.'],
```
Comment (perwestling) [inline lib/engine/game/g_18_bf/trains.rb:25]: More "minor companies" in this file.

File: lib/engine/game/g_18_ireland/entities.rb [hunk 25]
```diff
@@ -9,8 +9,8 @@ module Entities
             name: 'Dalkey Atmospheric Railway',
             value: 20,
             revenue: 5,
-            desc: 'No company can build in the Wicklow hex until this company is either bought by'\
-                  ' any company or closed.',
+            desc: 'No Corporation can build in the Wicklow hex until this company is either bought by'\
```
Comment (perwestling) [inline lib/engine/game/g_18_ireland/entities.rb:12]: Corporation -> corporation

File: lib/engine/game/g_18_nl/entities.rb [hunk 26]
```diff
@@ -10,7 +10,7 @@ module Entities
             sym: 'P1',
             value: 20,
             revenue: 0,
-            desc: 'Company may build an extra tile on F5 with a ƒ40 discount. Blocks F5 while owned by a player.',
+            desc: 'Owning corp may build an extra tile on F5 with a ƒ40 discount. Blocks F5 while owned by a player.',
```
Comment (perwestling) [inline lib/engine/game/g_18_nl/entities.rb:13]: corp -> corporation

File: lib/engine/game/g_18_usa/entities.rb [hunk 27]
```diff
@@ -164,13 +164,13 @@ module Entities
             name: 'P8 - Express Freight Service',
             value: 40,
             revenue: 0,
-            desc: 'Place an extra station marker from the owning company in one red area. The company receives +10 ' \
+            desc: 'Place an extra station marker from the owning corporation in one red area. The corp receives +10 ' \
```
Comment (perwestling) [inline lib/engine/game/g_18_usa/entities.rb:167]: Some "corp" in this file as well.

File: lib/engine/game/g_21_moon/entities.rb [hunk 28]
```diff
@@ -29,7 +29,7 @@ module Entities
             revenue: 5,
             min_price: 1,
             max_price: 45,
-            desc: 'When this private is bought by a company, the president of the company may choose to add or remove '\
+            desc: 'When this private is bought by a corporation, the president of the corporation may choose to add or remove '\
```
Comment (perwestling) [inline lib/engine/game/g_21_moon/entities.rb:32]: private -> company

?

**Comment by ollybh** [comment]
> @ollybh Many rulebooks use a lot of random terms for various entities. But at least on this site, isn't it better to have uniform terminology?

Do any rulebooks use the term 'corporation' for what are called corporations here? 1830 calls them 'railroads'. 1822 'major companies'. 1870 'public companies'. 1880 'share companies'.

18xx.games has four types of entities: company, minor, corporation and system. These often don't cleanly map to the types of entity in a game. Lots of games have minors that are implemented by corporations here (1861, 1867, all the 1822s), these should still be called minors even though they aren't represented by a minor object. Lots of games have multiple different types of privates and big companies, it would be better to use the names that the designer has chosen.

**Comment by ollybh** [comment]
> @ollybh 18xx.games has standardized the terminology

Do you think players are using this terminology? I had assumed that everyone still talks about privates and companies, not companies and corporations.

File: lib/engine/game/g_18_sj/entities.rb [hunk 29]
```diff
@@ -49,7 +49,7 @@ module Entities
             value: 45,
             revenue: 15,
             desc: 'May lay or shift port token in Halmstad (A6), Ystad(C2), Kalmar (D5), Sundsvall (F19), Umeå (F23), '\
-                  'and Luleå (G26).  Add 30 kr/symbol to all routes run to this location by owning company.',
+                  'and Luleå (G26).  Add 30 kr/symbol to all routes run to this location by owning corp.',
```
[the remark this answers is not in the document]
Comment (ollybh) [reply lib/engine/game/g_18_sj/entities.rb:52]: Agreed, if we are going to make this change it should always be 'corporation', never 'corp'.

File: lib/engine/game/g_18_ardennes/map.rb [hunk 30]
```diff
@@ -365,7 +365,7 @@ def after_lay_tile(hex, tile, entity)
         end
 
         def place_home_token(corporation)
-          # Public companies get their starting tokens by exchange.
+          # Corporations get their starting tokens by exchange.
```
Comment (ollybh) [inline lib/engine/game/g_18_ardennes/map.rb:368]: There's zero benefit in changing comments. Please leave these unchanged.

**Review by ollybh — CHANGES_REQUESTED** [review]
I'm not convinced about the need for this.
If it does go ahead, limit the changes to what is visible to players, do not change code comments. When I'm implementing a game, comments will be often be explaining how the code relates to the rulebook, so use the terminology from the rulebook.

File: lib/engine/game/g_18_ardennes/map.rb (hunk 30 above)
Comment (perwestling) [reply lib/engine/game/g_18_ardennes/map.rb:368]: I do not agree that it is zero benefits in changing comments.
For one thing it is a problem with comments that do not get updated when code change.
For another thing, internally (in the code) corporations, minors and companies are used, so when making the visible part updated there are no reason at all to keep the comment as is.

File: lib/engine/game/g_18_ardennes/market.rb (hunk 20 above)
Comment (philcampeau) [reply lib/engine/game/g_18_ardennes/market.rb:18]: I don't believe Toby established a protocol for minors.

File: lib/engine/game/g_18_ardennes/map.rb (hunk 30 above)
Comment (ollybh) [reply lib/engine/game/g_18_ardennes/map.rb:368]: 18Ardennes has two types of railway company: minor companies and public companies. These are both implemented as Corporation objects. This comment is explaining the different behaviour between the different type of companies, but if it is changed to read 'Corporations' then it is will be wrong. Some corporations get their starting tokens by exchange, some don't.

**Comment by scottredracecar** [comment]
I have been thinking about this more and it is bothering me. Please don’t
merge this.

If someone does a comprehensive review of game terms used in these game
rules/components and finds games that use the term corporation, those
could/should be changed.

But other games should not be changed to suit a backend system preference.

**Comment by roseundy** [comment]
I can't say I'm too crazy about this either.

There are several other terms that differ between 18xx.games code and what the game designer uses. For example 1860 and 1862 refer to "directors" instead of "presidents".

Personally, I think if we are going to change something like this, it should be in the code, and _not_ in the user interface. I actually thought about creating game-specific mappings so that the code could refer to "president" but be displayed to the user as "director". Ditto for "corporation" vs. "company" and other terms.

File: lib/engine/game/g_1817/entities.rb
Comment (benjaminxscott) [inline lib/engine/game/g_1817/entities.rb:1]: fwiw I think the changes in these 1817* files are valuable to merge independently for clarity

**Comment by philcampeau** [comment]
I understand the hesitation about this, but at the same time, you also end up with a lot of private company descriptions like this one: 

```
            desc: 'MAJOR/MINOR, Phase 2. Remove Small Station. Allows the owning company to place a plain yellow '\
                  'track tile directly on an undeveloped small station hex location or upgrade a small station tile '\
                  'of one colour to a plain track tile of the next colour. This closes the company and counts as '\
                  'the company’s normal track laying step. All other normal track laying restrictions apply. Once '\
                  'acquired, the private company pays its revenue to the owning company until the power is '\
                  'exercised and the company is closed.',
```

>This closes the company and counts as the company’s normal track laying step.

Do private companies have track laying steps? Oh 'company' means the owning company. Wait, does that mean my public company is going to close when I use the ability? Oh, 'company' means private company.

There's also the description of the Mail Contracts in 1817NA:

```
            desc: 'Pays owning corp $10 at the start of each operating round, as '\
                  'long as the company has at least one train.',
```

The terminology is already inconsistent throughout the site, and even in individual games. I'm mostly just proposing a clean-up of the language on the site. Corporation is used pretty much site-wide to mean share-corporations, minors are single share corporations, and private companies are privates. No one is confused when they go into the Auto tab and see 'Corporations' next to the list of auto buy options.

**Comment by scottredracecar** [comment]
So the proper fix is to change the text to say public company (or whatever
the game term is), not corporation. Adding a new game term is not right.

The spreadsheet should be changed to “Public Company” for several games to
make it match the rules if you are looking to add consistency.

**Comment by scottredracecar** [comment]
Also, directly responding to the item about 1817 specifically. Why should
that one introduce a new game term corporation? The rules don't call
anything corporations.

**Comment by philcampeau** [comment]
I don't feel like arguing this point. Never mind.

**Pull request closed without merging**